openwisp_users/models.py

```diff
@@ -35,6 +36,12 @@ class Meta(AbstractOrganizationOwner.Meta):
         swapper.swappable_setting('openwisp_users', 'OrganizationOwner')
 
 
+# only needed for django-organizations~=2.x
+class OrganizationInvitation(AbstractOrganizationInvitation):
+    class Meta(AbstractOrganizationInvitation.Meta):
+        abstract = True
```

Comment:
I don't think this should be abstract, I would allow this to be swappable since the start and update the README:
https://github.com/openwisp/openwisp-users#8-add-swapper-configurations

We should specify that right now we don't use this model in OpenWISP but users are free to implement custom flows with it if needed.

Some users may be interested in implementing this.

What do you think?